connect/hwinfo.go

```diff
@@ -0,0 +1,30 @@
+package connect
+
+import (
+	"bufio"
+	"bytes"
+	"strings"
+)
+
+func lscpu() (map[string]string, error) {
+	output, err := execute([]string{"lscpu"}, false, nil)
```

Comment:
`lscpu` has localized outputs. We should probably set some env variables to make sure we get expected output for parsing.